troposphere/emr.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2012-2013, Antonio Alonso Dominguez <alonso.domin@gmail.com>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSObject, AWSProperty, AWSHelperFn
+from .validators import (boolean, integer)
+
+
+class KeyValue(AWSHelperFn):
+    def __init__(self, key, value):
+        self.data = {'Key': key, 'Value': value}
+
+    def JSONrepr(self):
+        return self.data
+
+
+class Application(AWSProperty):
+    props = {
+        'AdditionalInfo': (dict, False),
```

Comment:
`AdditionalInfo` seems to be documented as a _JSON Object_, that's why I chose the `dict` to represent it. Is there a better way?

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-emr-cluster.html#cfn-emr-cluster-additionalinfo